spec/lib/embed/viewer/was_seed_spec.rb

```diff
@@ -19,12 +19,59 @@
+        "thumbnail_uri"=> "https://stacks.stanford.edu/image/iiif/gb089bd2251%2F20121129060351/full/200,/0/default.jpg"
+      },{
+        "memento_uri" => "https://swap.stanford.edu/20130412231301/http://naca.central.cranfield.ac.uk/",
+        "memento_datetime" => "20130412231301",
+        "thumbnail_uri" => "https://stacks.stanford.edu/image/iiif/gb089bd2251%2F20130412231301/full/200,/0/default.jpg"
+      }]
+      allow(was_seed_viewer).to receive(:thumbs_list).and_return(thumbs_list)
+      
+      html = Capybara.string(was_seed_viewer.to_html)
       # visible false because we display:none the container until we've loaded the CSS.
       expect(html).to have_css '.sul-embed-was-seed', visible: false
+      expect(html).to have_css '.sul-embed-was-thumb-list', visible: false, count: 1
+      expect(html).to have_css '.sul-embed-was-thumb-item', visible: false, count: 2
+      expect(html).to have_css '.sul-embed-was-thumb-item-div', visible: false, count: 2
+      
+      expect(html).to have_css '.sul-embed-was-thumb-item-div a[href="https://swap.stanford.edu/20121129060351/http://naca.central.cranfield.ac.uk/"]',  visible: false
+      expect(html).to have_css '.sul-embed-was-thumb-item-date',text: 'Nov 29, 2012', visible: false
+      expect(html).to have_css '.sul-embed-was-thumb-item img[src="https://stacks.stanford.edu/image/iiif/gb089bd2251%2F20121129060351/full/200,/0/default.jpg"]',  visible: false
+
+      expect(html).to have_css '.sul-embed-was-thumb-item-div a[href="https://swap.stanford.edu/20130412231301/http://naca.central.cranfield.ac.uk/"]',  visible: false
+      expect(html).to have_css '.sul-embed-was-thumb-item-date',text: 'Apr 12, 2013', visible: false
+      expect(html).to have_css '.sul-embed-was-thumb-item img[src="https://stacks.stanford.edu/image/iiif/gb089bd2251%2F20130412231301/full/200,/0/default.jpg"]',  visible: false
+    end
+  end
+  
+  describe 'thumbs_list' do
+    it 'calls the Embed::WasSeedThumbs with the same druid id' do
+      stub_request( request)
+      allow_any_instance_of(Embed::WasSeedThumbs).to receive(:get_thumbs_list)
+      expect(Embed::WasSeedThumbs).to receive(:new).with('12345').and_return(Embed::WasSeedThumbs.new('12345'))
+      was_seed_viewer.thumbs_list
     end
+    it 'calls the Embed::WasSeedThumbs with the same druid id' do
+      stub_request( request)
+      thumbs_list = [{
+        "memento_uri"=> "https://swap.stanford.edu/20121129060351/http://naca.central.cranfield.ac.uk/",
+        "memento_datetime"=> "20121129060351",
+        "thumbnail_uri"=> "https://stacks.stanford.edu/image/iiif/gb089bd2251%2F20121129060351/full/200,/0/default.jpg"
+      },{
+        "memento_uri" => "https://swap.stanford.edu/20130412231301/http://naca.central.cranfield.ac.uk/",
+        "memento_datetime" => "20130412231301",
+        "thumbnail_uri" => "https://stacks.stanford.edu/image/iiif/gb089bd2251%2F20130412231301/full/200,/0/default.jpg"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.